### PR TITLE
Adding License Key Instructions to Common Config

### PR DESCRIPTION
**Pull request by JoshTheHero**
Adding a section into common config doc to activate the license key per action item from a Q1 Support meeting.

### CODE REVIEW

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 1]
```diff
@@ -19,6 +19,18 @@ Create a second user with the admin role. This user can provide access should yo
 * Add a user with a known, valid email address and secure password within your organization.
 * Register that user with the FusionAuth application and the `admin` role.
 
+## Activate License Key
+
+Activating the license key will allow access to all of the features inside FusionAuth. The key is found in your account portal and is activated in the <strong>Reactor</strong> tab. Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. Non-Production may be used for any other instances such as Dev,QA, or Sandbox.
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:24]:
```suggestion
Activating the license key will allow access to all of the paid features inside FusionAuth. The key is found in your [account portal](https://account.fusionauth.io/account/plan/) and is activated in the <Breadcrumb>Reactor</Breadcrumb> tab. 

Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. 

Non-production keys may be used for any other instances such as your developer's laptops, QA, or UAT. 
```

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 2]
```diff
@@ -19,6 +19,18 @@ Create a second user with the admin role. This user can provide access should yo
 * Add a user with a known, valid email address and secure password within your organization.
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:19]: On line 1 of this file, please add

import Breadcrumb from 'src/components/Breadcrumb.astro';

And make sure this renders correctly.

**Review by mooreds — CHANGES_REQUESTED** [review]
A few wording changes and using a different component for markup.

Great contribution, thanks!

File: astro/src/content/docs/_shared/_common-configuration.mdx (hunk 2 above)
Comment (JoshTheHero) [reply astro/src/content/docs/_shared/_common-configuration.mdx:20]: Added in there, looks normal on my end.

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 3]
```diff
@@ -19,6 +20,22 @@ Create a second user with the admin role. This user can provide access should yo
...
+## Activate License Key
+
+Activating the license key will allow access to all of the paid features inside FusionAuth. The key is found in your [account portal](https://account.fusionauth.io/account/plan/) and is activated in the <Breadcrumb>Reactor</Breadcrumb> tab. 
+
+Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. 
+
+Non-production keys may be used for any other instances such as your developer's laptops, QA, or UAT. 
+
+* Log in to the administrative user interface.
+* Navigate to <strong>Plan</strong> and copy the appropiate key
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:32]:
```suggestion
* Navigate to <strong>Plan</strong> and copy the appropriate key.
```

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 4]
```diff
@@ -19,6 +20,22 @@ Create a second user with the admin role. This user can provide access should yo
...
+* Access the FusionAuth instance and navigate to the <strong>Reactor</strong>
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:33]:
```suggestion
* Access the FusionAuth instance and navigate to the <strong>Reactor</strong>.
```

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 5]
```diff
@@ -19,6 +20,22 @@ Create a second user with the admin role. This user can provide access should yo
...
+Activating the license key will allow access to all of the paid features inside FusionAuth. The key is found in your [account portal](https://account.fusionauth.io/account/plan/) and is activated in the <Breadcrumb>Reactor</Breadcrumb> tab. 
+
+Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. 
+
+Non-production keys may be used for any other instances such as your developer's laptops, QA, or UAT. 
+
+* Log in to the administrative user interface.
+* Navigate to <strong>Plan</strong> and copy the appropiate key
+* Access the FusionAuth instance and navigate to the <strong>Reactor</strong> 
+* Paste that key into the License Key field and Activate it
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:34]:
```suggestion
* Paste that key into the License Key field and Activate it.
```

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 6]
```diff
@@ -19,6 +20,22 @@ Create a second user with the admin role. This user can provide access should yo
...
+
+[Full instructions on how to activate your license ](/docs/get-started/core-concepts/licensing)
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:36]:
```suggestion
[Full instructions on how to activate your license](/docs/get-started/core-concepts/licensing).
```

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 7]
```diff
@@ -19,6 +20,22 @@ Create a second user with the admin role. This user can provide access should yo
...
+
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:37]:
```suggestion
```

**Review by mooreds — COMMENTED** [review]
A few other small changes.

File: astro/src/content/docs/_shared/_common-configuration.mdx (hunk 4 above)
Comment (mooreds) [reply astro/src/content/docs/_shared/_common-configuration.mdx:33]: breadcrumb is for all navigation.

File: astro/src/content/docs/_shared/_common-configuration.mdx (hunk 3 above)
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:32]:
```suggestion
* Navigate to <Breadcrumb>Plan</Breadcrumb> and copy the appropiate key
```

File: astro/src/content/docs/_shared/_common-configuration.mdx [hunk 8]
```diff
@@ -19,6 +20,22 @@ Create a second user with the admin role. This user can provide access should yo
 * Add a user with a known, valid email address and secure password within your organization.
 * Register that user with the FusionAuth application and the `admin` role.
 
+## Activate License Key
+
+Activating the license key will allow access to all of the paid features inside FusionAuth. The key is found in your [account portal](https://account.fusionauth.io/account/plan/) and is activated in the <Breadcrumb>Reactor</Breadcrumb> tab. 
+
+Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. 
+
+Non-production keys may be used for any other instances such as your developer's laptops, QA, or UAT.
```
Comment (mooreds) [inline astro/src/content/docs/_shared/_common-configuration.mdx:29]:
```suggestion
Non-production keys may be used for any other instances such as your developers' laptops, QA, or UAT. 
```

File: astro/src/content/docs/_shared/_common-configuration.mdx (hunk 3 above)
Comment (JoshTheHero) [reply astro/src/content/docs/_shared/_common-configuration.mdx]: manually added

**Pull request merged**